src/nxt_controller.c

```diff
@@ -2466,6 +2467,17 @@ nxt_controller_conf_store(nxt_task_t *task, nxt_conf_value_t *conf)
                                 NXT_PORT_MSG_CONF_STORE | NXT_PORT_MSG_CLOSE_FD,
                                  fd, 0, -1, b);
 
+    rt->conf_gen++;
+
+    time(&rawtime);
+    timeinfo = gmtime(&rawtime);  //convert to UTC
+    strftime((char*)buffer, 25, "%Y-%m-%dT%H:%M:%S.000Z", timeinfo);
+
+    rt->conf_ltime.length = strlen((char*)buffer);
+    rt->conf_ltime.start = (u_char*) malloc(rt->conf_ltime.length + 1);
```

Comment:
We don't tend to call malloc(3) directly. There is a nxt_malloc() you can use.

Also please don't cast the return, in C it simply isn't required and (in that past at least) can hide bugs.